Sources/NetworkProtection/Diagnostics/NetworkProtectionConnectionTester.swift

```diff
@@ -267,9 +267,8 @@ final class NetworkProtectionConnectionTester {
         if failureCount == 0 {
             resultHandler(.connected)
         } else if failureCount > 0 {
+            resultHandler(.reconnected(failureCount: failureCount))
```

Comment:
We want to know how many failures there were before recovery, as we want to send different recovery pixels depending on it.